modules/registry-common-terms.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * registry/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-registry-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+container::
+Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers anywhere, from a data center to a public or private cloud to your local host.
+
+Image Registry Operator::
+The Image Registry Operator runs in the `openshift-image-registry` namespace, and manages the registry instance in that location.
+
+image repository::
+An image repository is a collection of related container images and tags identifying images.
+
+mirror registry::
+A registry that holds the mirror of {product-title} images.
+
+namespace::
+A namespace isolates specific system resources that are visible to all processes. Inside a namespace, only processes that are members of that namespace can see those resources.
+
+{product-title} registry::
+{product-title} registry is a registry provided by {product-title} to manage images.
+
+pod::
+The pod is the smallest logical unit in Kubernetes. A pod is comprised of one or more containers to run in a worker node.
+
+private registry::
+{product-title} can use any server implementing the container image registry API as a source of the image which allows the developers to push and pull their private container images.
+
+public registry::
+{product-title} can use any server implementing the container image registry API as a source of the image which allows the developers to push and pull their public container images.
```

Comment:
This section confused me a bit, so I gave a stab at clarifying it - feel free to use it as is or as inspiration.

```suggestion
private registry::
A registry is a server that implements the container image registry API. A private registry is a registry that requires authentication to allow users access to its contents.

public registry::
A registry is a server that implements the container image registry API. A public registry is a registry that serves its contently publicly.
```

Reply:
acked